orbstation/quirks/augmented.dm

```diff
@@ -0,0 +1,17 @@
+/datum/quirk/augmented
+	name = "Fully Augmented"
+	desc = "You never asked for this. Or maybe you did. Either way, all your limbs are cybernetic."
+	icon = "bolt-auto"
+	value = 0
+	medical_record_text = "During physical examination, patient was found to have fully augmented limbs."
```

Comment:
This isn't a suggestion that it should change (and it's on the base quirks), the wording here is just very funny.
"On close examination it turns out the patient's obviously metal limbs are in fact made of metal".

Reply:
you say this, but the number of times someone has nearly od'd us because they didn't check our limbs,,,,